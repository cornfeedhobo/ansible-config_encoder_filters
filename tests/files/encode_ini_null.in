var1: null
var2:
var3: "!!!null"

section1:
  var1: null
  var2:
  var3: "!!!null"
