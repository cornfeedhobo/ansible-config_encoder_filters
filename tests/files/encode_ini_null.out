var3

[section1]
var3
